### ISSUE TITLE
OSError: Error: Trying to run a BRAMA Target over NO-BRAMA compile.

### ISSUE THREAD

**Issue by greensquirrel** [ANR-COMPASS/shesha#1]
Dear COMPASS team!

I have tried your instructions from the README file.  Installation works fine but
the GUI fails to start with the following result:

```
[compass_github@nicelab shesha]$ ipython -i test/closed_loop.py data/par/par4bench/scao_sh_16x16_8pix.py
Python 3.6.2 |Anaconda, Inc.| (default, Sep 30 2017, 18:42:57) 
Type 'copyright', 'credits' or 'license' for more information
IPython 6.1.0 -- An enhanced Interactive Python. Type '?' for help.
/home/compass_github/shesha/src/shesha_init/lgs_init.py:14: UserWarning: 'SHESHA_DB_ROOT' not defined, using default one: /home/compass_github/shesha/data/
  warnings.warn("'SHESHA_DB_ROOT' not defined, using default one: " + shesha_db)
shesha_savepath: /home/compass_github/shesha/data/
('loading ', 'scao_sh_16x16_8pix')
->tel
*-----------------------
Computing geometry of WFS 0
r0 for WFS : 0.16  m
seeing for WFS :  0.64 "
quantum pixsize :  0.0580 "
simulated FoV :  3.71 " x  3.71 "
actual pixsize :  0.2901
actual FoV :  2.32 " x  2.32 "
number of phase points :  18
size of fft support :  64
size of HR spot support :  64
Generic pupil created
nphotons :  1577393.3612004826
->atmos
r0^-5/6 :0.0184971
create stencil and Z,X matrices
create zz
create xz
create xx
compute zz pseudo_inverse
compute A
compute bbt
svd of bbt
compute B
->dm
Pattern type : square
Computing Influence Function type :  b'default'
100%|████████████████████████████████████████| 284/284 [00:00<00:00, 636.76it/s]
100%|██████████████████████████████| 124372/124372 [00:00<00:00, 1359220.76it/s]
->target
---------------------------------------------------------------------------
OSError                                   Traceback (most recent call last)
~/shesha/test/closed_loop.py in <module>()
     32 
     33 sim.config.p_controller0.set_type(ControllerType.GENERIC)
---> 34 sim.init_sim()
     35 sim.loop(sim.config.p_loop.niter)

~/shesha/src/shesha_sim/simulator.py in init_sim(self)
    199             self.dms = None
    200 
--> 201         self._tar_init()
    202 
    203         if self.config.p_wfss is not None:

~/shesha/src/shesha_sim/simulatorBrama.py in _tar_init(self)
     20                                         self.config.p_atmos, self.config.p_tel,
     21                                         self.config.p_geom, self.config.p_dms,
---> 22                                         brama=True)
     23         else:
     24             self.tar = None

~/shesha/src/shesha_init/target_init.py in target_init(ctxt, telescope, p_target, p_atmos, p_tel, p_geom, dm, brama)
     63             target = Target_brama(ctxt, telescope, p_target.ntargets, p_target.xpos,
     64                                   p_target.ypos, p_target.Lambda, p_target.mag,
---> 65                                   p_target.zerop, sizes, Npts)
     66         else:
     67             target = Target(ctxt, telescope, p_target.ntargets, p_target.xpos,

src/sutra_bind/Target.pyx in Target.Target_brama.__cinit__()

OSError: Error: Trying to run a BRAMA Target over NO-BRAMA compile.

In [1]: 
```

Any ideas?

Thank you in advance!

**Comment by a-sevin** [comment]
Hi, 
I'm sorry, it was a dirty hack that shouldn't be committed
fixed @ 909755b810cbcae92cc339792b5a6c3bb66e081a
Arnaud

**Comment by greensquirrel** [comment]
It works!

```
[...]

100%|██████████████████████████████| 100468/100468 [00:00<00:00, 1360118.70it/s]
Done
controller uses 1 streams
Doing imat...100%
done in 0.333715 s
Doing imat svd...
svd done in 0.071529 s
Building cmat...
Filtering  18  modes
cmat done in 0.000761 s
----------------------------------------------------
iter# | S.E. SR | L.E. SR | ETR (s) | Framerate (Hz)
----------------------------------------------------
100 	 0.715 	  0.715	     1.0 	 874.1
200 	 0.732 	  0.724	     0.9 	 872.0
300 	 0.572 	  0.673	     0.8 	 871.1
400 	 0.723 	  0.686	     0.7 	 870.8
500 	 0.622 	  0.673	     0.6 	 870.8
600 	 0.777 	  0.690	     0.5 	 871.0
700 	 0.702 	  0.692	     0.3 	 871.0
800 	 0.757 	  0.700	     0.2 	 870.9
900 	 0.598 	  0.689	     0.1 	 870.8
1000 	 0.705 	  0.690	     0.0 	 870.7
 loop execution time: 1.1499271392822266   ( 1000 iterations),  0.0011499271392822266 (mean)   869.6203140524107 Hz

In [1]: 

```
